src/utils/config.rs

```diff
@@ -96,7 +98,7 @@ impl Config {
         DirBuilder::new()
             .recursive(true)
             .create(Path::new(&path))
-            .unwrap();
+            .expect("error creating directory.");
```

Comment:
Change to Sentence Case

Reply:
plan is to make a gracious error window instead but i am still researching iced